frontend/src/components/Admin/CoffeeChats/CoffeeChatDetails.tsx

```diff
@@ -85,10 +99,6 @@ const CoffeeChatDetails: React.FC = () => {
   const [status, setStatus] = useState<Status | string>();
   const [coffeeChats, setCoffeeChats] = useState<CoffeeChat[]>([]);
 
-  useEffect(() => {
```

Comment:
This is a duplicate call. If you look further down, we already fetch all coffee chats.